bingo/oracle/src/oracle/mango_oracle.cpp

```diff
@@ -16,12 +16,11 @@
  * limitations under the License.
  ***************************************************************************/
 
-#include "oracle/ora_wrap.h"
-#include "oracle/mango_oracle.h"
 #include <memory>
-#include "base_cpp/output.h"
+
 #include "oracle/bingo_oracle_context.h"
-#include "oracle/mango_shadow_table.h"
```

Comment:
Please check: 2 include files were lost.

Reply:
after reordering some includes became redundant and so've been deleted.